google/cloud/storage/oauth2/google_application_default_credentials_file.h

```diff
@@ -23,11 +23,37 @@ namespace cloud {
 namespace storage {
 inline namespace STORAGE_CLIENT_NS {
 namespace oauth2 {
-/// Return the path for the default service account credentials file.
-std::string GoogleApplicationDefaultCredentialsFile();
 
-/// The name of the environment variable to configure `HOME`.
-char const* GoogleApplicationDefaultCredentialsHomeVariable();
+/**
+ * Returns the environment variable that should be checked for a valid file
+ * path when attempting to load Google Application Default Credentials.
+ */
+inline char const* GoogleAdcEnvVar() {
+  static constexpr char kEnvVarName[] = "GOOGLE_APPLICATION_CREDENTIALS";
+  return kEnvVarName;
+}
+
+/**
+ * Returns the path to the file containing Application Default Credentials, if
```

Comment:
Consider splitting this into a short description and a longer description.  The doxygen documentation has summaries based on the one-liner at the beginning of the comment block.

Reply:
Fixed.